onboarding/source/controller.c

```diff
@@ -122,6 +143,13 @@ static void ledTimerCallback(TimerHandle_t xTimer) {
 static void lightTimerCallback(TimerHandle_t xTimer) {
     /* USER CODE BEGIN */
     // Send light event to light service queue
+    ASSERT(xTimer != NULL);
+    light_event_t * measureLightPtr = MEASURE_LIGHT;
+    obc_error_code_t lightTimerReturn = sendToLightServiceQueue(measureLightPtr);
```

Comment:
`light_event_t * measureLightPtr = MEASURE_LIGHT;` would create a pointer to address `0x0` (since MEASURE_LIGHT = 0). This isn't exactly what we want. We want to create a variable of type `light_event_t` with a  value of `MEASURE_LIGHT`. `sendToLightServiceQueue` expects a pointer to this variable.